portal-impl/src/com/liferay/portlet/documentlibrary/model/impl/DLFileEntryImpl.java

```diff
@@ -245,6 +245,10 @@ public Lock getLock() {
 			DLFileEntry.class.getName(), getFileEntryId());
 	}
 
+	/**
```

Comment:
@adolfopa I decided to deprecate it cause is only used on the contributor.
but I have not a strong opinion about it :D